subsys/bluetooth/controller/ll_sw/ull_llcp.c

```diff
@@ -278,7 +281,7 @@ struct proc_ctx *llcp_create_local_procedure(enum llcp_proc proc)
 {
 	struct proc_ctx *ctx;
 
-	ctx = create_procedure(proc);
+	ctx = create_procedure(proc, &mem_ctx);
```

Comment:
General question, not related with this PR itself.
I think that there should be some naming convention for globals to make them easier to distinguish from local variables and function arguments. What do you think?